plugins/circuitbreaker/README.md

```
@@ -0,0 +1,209 @@
+                    Role: schemas.ModelChatMessageRoleUser,
+                    Content: schemas.MessageContent{
+                        ContentStr: bifrost.Ptr("Hello!"),
+                    },
+                },
+            },
+        },
+    })
+}
+```
+
+### State Diagram of Circuit Breaker
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)